docs/proposals/25-private-cloud-profiles.md

```diff
@@ -0,0 +1,299 @@
+---
```

Comment:
Nit: you could add the GEP to the index. See https://github.com/gardener/gardener/pull/9170

Reply:
Done, thanks.